### PR TITLE
Color Temperature value update fix for #1318

### PR DESCRIPTION
**Pull request by shry15harsh**

### CODE REVIEW

**Comment by maggu2810** [comment]
Thanks for your contribution.
Please drop the changes of the file "CONTRIBUTING.md" from your branch and squash your commits.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,3 +44,4 @@ Here is a sample _good_ Git commit log message:
 
 Finally, a contribution is not a good contribution unless it comes with unit tests, integration tests and
```
Comment (kaikreuzer) [inline CONTRIBUTING.md]: Would it make sense to add test cases for the LightStateConverter to make sure that it works as expected?

Comment (maggu2810) [inline CONTRIBUTING.md]: Is this a rhetorical question? IMHO a test make sense in the most cases, here too.

**Comment by kaikreuzer** [comment]
@shry15harsh Could you please add some unit test for this? Would be lovely :-)

**Comment by shry15harsh** [comment]
@kaikreuzer I'll do it. :)

**Comment by shry15harsh** [comment]
@kaikreuzer Please check. Let me know if it needs some refactoring or any change.

**Comment by kaikreuzer** [comment]
Cool, thanks you!

**Pull request merged**